gapis/api/vulkan/resources.go

```diff
@@ -1197,9 +1197,23 @@ func (p GraphicsPipelineObjectʳ) geometryShader() *api.Stage {
 }
 
 func (p GraphicsPipelineObjectʳ) rasterizer() *api.Stage {
+	rasterState := p.RasterizationState()
+	rasterList := &api.KeyValuePairList{}
+	rasterList = rasterList.AppendKeyValuePair("Depth Clamp Enabled", api.CreatePoDDataValue("VkBool32", rasterState.DepthClampEnable() != 0))
+	rasterList = rasterList.AppendKeyValuePair("Rasterizer Discard", api.CreatePoDDataValue("VkBool32", rasterState.RasterizerDiscardEnable() != 0))
+	rasterList = rasterList.AppendKeyValuePair("Polygon Mode", api.CreateEnumDataValue("VkPolygonMode", rasterState.PolygonMode()))
+	rasterList = rasterList.AppendKeyValuePair("Cull Mode", api.CreateBitfieldDataValue("VkCullModeFlags", rasterState.CullMode(), VkCullModeFlagBitsConstants(), api.Find(api.ID(ID))))
```

Comment:
`api.Find(api.ID(ID))` is the same as `API{}` (see the generated `api.go` file for each API).

Reply:
Done. I would still need to pass it to the function no matter what, correct?